terragrunt/common/common_variables.tf

```
@@ -24,3 +24,13 @@ variable "secondary_location" {
   description = "Sets the location for \"secondary\" resources to be created in."
   default     = "canadaeast"
 }
+
+variable "SECURITY_EMAIL" {
+  description = "The email address for the security resolver group"
+  type        = string
+}
+
+variable "SRE_EMAIL" {
+  description = "The email address for the SRE resolver group"
+  type        = string
+}
```

Just for consistency, I would have these as lowercase to match the other variable casing:
```suggestion
variable "security_email" {
  description = "The email address for the security resolver group"
  type        = string
}

variable "sre_email" {
  description = "The email address for the SRE resolver group"
  type        = string
}
```